lara-plugin-api/src/api/plugins.ts

```diff
@@ -128,27 +55,21 @@ export const initPlugin = (label: string, runtimeContext: IRuntimeContext, plugi
  @param pluginId ID of the plugin trying to save data, initially passed to plugin constructor in the context.
  @param state A JSON string representing serialized plugin state.
```

Comment:
It would be useful to put a note about how the plugin scope plays a role here. If the plugin is at the activity level then this state should be shared across all pages. At the page and item level it is just for that page or item.